Jenkinsfile

```diff
@@ -42,5 +67,23 @@ pipeline{
 				}
 			}
 		}
+		// Logs and data files generated by this step are archived in the Reactome S3 bucket.
+		// All files are then deleted on server.
+		stage('Post: Archive Outputs'){
+			steps{
+				script{
+					def s3Path = "${env.S3_RELEASE_DIRECTORY_URL}/${currentRelease}/orthopairs"
+					sh "mkdir -p data/orthopairs/"
+					sh "mv *alternate_ids.txt data/"
+					sh "mv ${currentRelease}/* data/orthopairs/"
+					sh "mv *.gz data/"
+					sh "gzip -r logs/ data/"
```

Comment:
Should the gzip command be before the command for moving *.gz files?  If you move gzipped files to the data directory and then gzip the directory, it's likely to output warnings for the files that are already present and gzipped from the move.

Reply:
Right, I even noticed that when I was doing this and then forgot